src/hypervisor.cc

```diff
@@ -291,16 +291,16 @@ NLV_WORKER_EXECUTE(Hypervisor, Connect)
 
 NAN_METHOD(Hypervisor::Disconnect)
 {
-  NanScope();
-  if (args.Length() == 1 && !args[0]->IsFunction()) {
-    NanThrowTypeError("You must specify a function as first argument");
-    NanReturnUndefined();
+  Nan::HandleScope scope;
+  if (info.Length() == 1 && !info[0]->IsFunction()) {
+    Nan::ThrowTypeError("You must specify a function as first argument");
+    return;
   }
 
-  NanCallback *callback = new NanCallback(args[0].As<Function>());
-  Hypervisor *hv = ObjectWrap::Unwrap<Hypervisor>(args.This());
-  NanAsyncQueueWorker(new DisconnectWorker(callback, hv));
-  NanReturnUndefined();
+  Nan::Callback *callback = new Nan::Callback(info[0].As<Function>());
+  Hypervisor *hv = ObjectWrap::Unwrap<Hypervisor>(info.This());
+  Nan::AsyncQueueWorker(new DisconnectWorker(callback, hv));
+  return;
```

Comment:
To be honest I wasn't thinking too much about purpose of the code so I'd rather just move all of the code to nan2, and think about code elimination as part of different pull requests. :)